src/rest/Plan.ts

```diff
@@ -30,3 +30,23 @@ export interface IPlan {
     largeExpression: string;
   };
 }
+
+/**
+ * Describes the plan of execution of a search
+ */
+export class ExecutionPlan {
+  constructor(private response: IPlanResponse) {}
+
+  public get basicExpression() {
+    return this.response.parsedInput.basicExpression;
+  }
+
+  public get largeExpression() {
```

Comment:
Suggestion:

Gets the final value of the large expression (`lq`) after the search request has been processed in the query pipeline, but before it is sent to the index.